modular_skyrat/modules/loadouts/loadout_items/loadout_datum_toys.dm

```diff
@@ -291,3 +291,8 @@ GLOBAL_LIST_INIT(loadout_toys, generate_loadout_items(/datum/loadout_item/toys))
 /datum/loadout_item/toys/crayons
 	name = "Box of crayons"
 	item_path = /obj/item/storage/crayons
+
+/datum/loadout_item/toys/xixi
```

Comment:
Personal opinion: add this next to the other plushies

Reply:
fair note actually cheers